src/__swaps__/screens/Swap/resources/assets/userAssetsByChain.ts

```diff
@@ -90,24 +91,3 @@ export async function userAssetsByChainQueryFunction({
 }
 
 type UserAssetsByChainResult = QueryFunctionResult<typeof userAssetsByChainQueryFunction>;
-
-// ///////////////////////////////////////////////
-// Query Hook
-
-export function useUserAssetsByChain<TSelectResult = UserAssetsByChainResult>(
```

Comment:
deleting since this wasn't used anywhere

Reply:
same with this